FindBaselibs.cmake

```diff
@@ -1,10 +1,67 @@
-set (BASEDIR "" CACHE PATH "Path to installed baselibs _including_ OS subdirectory (Linux or Darwin).")
+set (BASEDIR "" CACHE PATH "Path to installed baselibs")
 set (Baselibs_FOUND FALSE)
```

Comment:
Should this be a cached value?

Reply:
Huh. I don't know. I mean, I guess I kept it that way because it was that way before and working? 

Before I changed it, the line was attributed to @rmontuoro and before that our initial commit.

I know what is here seems to work. Once it is merged, I can then test out if it's not CACHE'd?

Reply:
Sorry clicked on the wrong line.  I meant the Baselibs_FOUND variable.  That should be cached.

Reply:
And then technically, the rest of the logic should be skipped if it is already TRUE (avoids some overhead with cmake reprocessing, though here that would be very slight)